### PR TITLE
Improve/update

### PR DESCRIPTION
**Pull request by ThinkTankShark**

### CODE REVIEW

**Comment by ThinkTankShark** [comment]
@isuttell @corescan Please let me know if you are moving forward with this component so I can fix the tests to pass, or delete the repo. Thanks

File: src/__tests__/ProgressCircle-test.jsx [hunk 1]
```diff
@@ -55,9 +55,5 @@ describe('ProgressCircle', function() {
       );
       let comp = TestUtils.findRenderedComponentWithType(reactTree, ProgressCircle);
       expect(comp.state.pathLength).toEqual(circumference);
-
-      let path = TestUtils.findRenderedDOMComponentWithClass(reactTree, 'progress-circle--path');
-      expect(parseFloat(path._style.strokeDashoffset)).toBeCloseTo(dashOffset, 6);
```
Comment (isuttell) [inline src/__tests__/ProgressCircle-test.jsx]: Why was this test removed?

File: tasks/config/eslint.js [hunk 2]
```diff
@@ -10,6 +10,9 @@
  */
 module.exports = function(grunt) {
   grunt.config.set('eslint', {
-    target: ['src/**/*.+(jsx?|es6)']
+    target: ['src/**/*.jsx'],
```
Comment (isuttell) [inline tasks/config/eslint.js]: Let's switch to the other glob so we get js files as well

Comment (ThinkTankShark) [inline tasks/config/eslint.js]: Fixed

File: src/__tests__/ProgressCircle-test.jsx (hunk 1 above)
Comment (ThinkTankShark) [inline src/__tests__/ProgressCircle-test.jsx]: This test was throwing an 'undefined' error. Probably a syntax change or something that sort with the new version of Jest or React. I will dedicate a sprint or half in future to add more tests to all of our components.

**Pull request merged**